spec/index.bs

```diff
@@ -7058,13 +7067,15 @@ enum GPUStoreOp {
 
             Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
             <div class=device-timeline>
+                1. Let |pipelineEncoderDesc| be [$derive render encoder descriptor from pipeline$](|pipeline|.{{GPURenderPipeline/[[descriptor]]}}).
                 1. If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
                     <div class=validusage>
                         - |pipeline| is [$valid to use with$] |this|.
-
-                        Issue: Validate that |pipeline| is compatible with the render pass descriptor.
+                        - |this|.{{GPURenderEncoderBase/[[descriptor]]}} equals |pipelineEncoderDesc|.
```

Comment:
note: Unfortunately there is no [=ordered map/equal=] or similar. Could add something like `(the unordered sets of their key-value pairs are equal)` but that doesn't seem that necessary.

Reply:
we could add an issue to define what the "equals" mean here